code/modules/projectiles/guns/projectile/automatic/molly.dm

```diff
@@ -1,6 +1,10 @@
 /obj/item/weapon/gun/projectile/automatic/molly
 	name = "FS MP .35 Auto \"Molly\""
-	desc = "An experimental fully automatic pistol. Compact and flexible, but somewhat underpowered. Issued to non-combatants among Ironhammer as powerful self-protection sidearm. Custom magwell allows it to feed both from highcap pistol and SMG magazines. Uses .35 Auto rounds."
+	desc = "An experimental fully automatic pistol. Was designed as a middle ground between SMGs and pistols to put superior \
```

Comment:
"An experimental fully automatic pistol, designed as a middle ground between SMGs and Pistols. Primarily employed in CQC scenarios or as a civilian self defence tool. Takes both .35 Highcap and .35 Auto rounds."

Reply:
Used it, thanks.